2024-04-15 16:14:37,301 py30 INFO my_logger.py 56行：测试，我自己封装的日志类！
2024-04-15 16:15:41,259 py30 INFO my_logger.py 56行：测试，我自己封装的日志类！
2024-04-15 16:19:41,589 py30 INFO handle_requests.py 58行：发起一次HTTP请求
2024-04-15 16:19:41,589 py30 INFO handle_requests.py 66行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 16:19:41,589 py30 INFO handle_requests.py 67行：请求方法为：POST
2024-04-15 16:19:41,590 py30 INFO handle_requests.py 68行：请求url为：http://192.168.66.67/ykt/http://192.168.66.67/ykt/token/get
2024-04-15 16:19:41,591 py30 INFO handle_requests.py 69行：请求数据为：{'loginName': 'admin', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b'}
2024-04-15 16:19:41,676 py30 INFO handle_requests.py 76行：响应状态码为：200
2024-04-15 16:19:41,677 py30 INFO handle_requests.py 77行：响应数据为：{'msg': 'token不存在', 'status': 2}
2024-04-15 16:25:59,572 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 16:25:59,573 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 16:25:59,573 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 16:25:59,573 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/http://192.168.66.67/ykt/token/get
2024-04-15 16:25:59,573 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b'}
2024-04-15 16:25:59,650 py30 INFO handle_requests.py 63行：响应状态码为：200
2024-04-15 16:25:59,650 py30 INFO handle_requests.py 64行：响应数据为：{'msg': 'token不存在', 'status': 2}
2024-04-15 16:28:24,558 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 16:28:24,559 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 16:28:24,559 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 16:28:24,559 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/http://192.168.66.67/ykt/token/get
2024-04-15 16:28:24,559 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b'}
2024-04-15 16:35:29,159 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 16:35:29,159 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 16:35:29,160 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 16:35:29,160 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 16:35:29,160 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b'}
2024-04-15 16:36:37,089 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 16:36:37,089 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 16:36:37,089 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 16:36:37,089 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 16:36:37,089 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b'}
2024-04-15 16:36:37,161 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 16:36:37,161 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8', 'accessToken': 'eyJhbGciOiJIUzI1NiJ9.eyJqdGkiOiIxIiwic3ViIjoiYWRtaW4iLCJpc3MiOiJEUlpLIiwiaWF0IjoxNzEzMTcwMTYxLCJleHAiOjE3MTMyMzQ5NjF9.tenwn185GX_qWFlMQjopDyKr8XiJpC0xdhmQULKlx4Q'}
2024-04-15 16:36:37,161 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 16:36:37,161 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/api/v1/visitor/create
2024-04-15 16:36:37,162 py30 INFO handle_requests.py 56行：请求数据为：{'carNo': '粤B121981', 'startTime': '2024-09-14 11:11:11', 'endTime': '2024-09-17 11:11:11', 'visitName': '张三', 'visitorPhone': '19852461254', 'visitorReason': '备注', 'dsn': '145456784321,456451544653', 'objectId': 15}
2024-04-15 16:39:35,356 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 16:39:35,357 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 16:39:35,357 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 16:39:35,357 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 16:39:35,357 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b'}
2024-04-15 16:39:35,394 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 16:39:35,394 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8', 'accessToken': 'eyJhbGciOiJIUzI1NiJ9.eyJqdGkiOiIxIiwic3ViIjoiYWRtaW4iLCJpc3MiOiJEUlpLIiwiaWF0IjoxNzEzMTcwMzQwLCJleHAiOjE3MTMyMzUxNDB9.r_LfsN_4sRLUhYbB5eaplz9ogJV9_mPFpOtIrburFdc'}
2024-04-15 16:39:35,394 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 16:39:35,394 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/api/v1/visitor/create
2024-04-15 16:39:35,394 py30 INFO handle_requests.py 56行：请求数据为：{'carNo': '粤B921981', 'startTime': '2024-09-14 11:11:11', 'endTime': '2024-09-17 11:11:11', 'visitName': '张三', 'visitorPhone': '19852461254', 'visitorReason': '备注', 'dsn': '145456784321,456451544653', 'objectId': 16}
2024-04-15 18:50:02,764 py30 INFO test_login.py 48行：======= 登录模块用例  开始执行========
2024-04-15 18:50:02,766 py30 INFO test_login.py 57行：执行用例1：正常登录
2024-04-15 18:50:02,766 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 18:50:02,766 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 18:50:02,767 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 18:50:02,767 py30 INFO handle_requests.py 55行：请求url为：/token/get
2024-04-15 18:50:02,767 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b"
}
2024-04-15 18:50:02,789 py30 INFO test_login.py 57行：执行用例2：name为空
2024-04-15 18:50:02,789 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 18:50:02,789 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 18:50:02,789 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 18:50:02,789 py30 INFO handle_requests.py 55行：请求url为：/token/get
2024-04-15 18:50:02,790 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8942b"
}
2024-04-15 18:50:02,794 py30 INFO test_login.py 57行：执行用例3：word为空
2024-04-15 18:50:02,794 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 18:50:02,794 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 18:50:02,794 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 18:50:02,794 py30 INFO handle_requests.py 55行：请求url为：/token/get
2024-04-15 18:50:02,794 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":""
}
2024-04-15 18:50:02,799 py30 INFO test_login.py 57行：执行用例4：name错误
2024-04-15 18:50:02,799 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 18:50:02,799 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 18:50:02,799 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 18:50:02,800 py30 INFO handle_requests.py 55行：请求url为：/token/get
2024-04-15 18:50:02,800 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin1",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b"
}
2024-04-15 18:50:02,805 py30 INFO test_login.py 57行：执行用例5：word错误
2024-04-15 18:50:02,805 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 18:50:02,805 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 18:50:02,805 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 18:50:02,805 py30 INFO handle_requests.py 55行：请求url为：/token/get
2024-04-15 18:50:02,805 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b1"
}
2024-04-15 18:50:02,810 py30 INFO test_login.py 52行：======= 登录模块用例  执行结束========
2024-04-15 18:53:41,554 py30 INFO test_login.py 49行：======= 登录模块用例  开始执行========
2024-04-15 18:53:41,554 py30 INFO test_login.py 58行：执行用例1：正常登录
2024-04-15 18:53:41,554 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 18:53:41,554 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 18:53:41,554 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 18:53:41,554 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 18:53:41,555 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b"
}
2024-04-15 18:53:41,973 py30 INFO test_login.py 66行：用例的期望结果为：{
    "status": 1,
    "msg": "登录成功",
    "data": {
        "token": "eyJhbGciOiJIUzI1NiJ9.eyJqdGkiOiIxIiwic3ViIjoiYWRtaW4iLCJpc3MiOiJEUlpLIiwiaWF0IjoxNzEyMDI4OTA3LCJleHAiOjE3MTIwOTM3MDd9.LBQLzIE9jovM4md8psL46eV4J7M9NhHEIx5pyekqcuc",
        "loginStatus": "2"
    }
}
2024-04-15 18:53:41,974 py30 ERROR test_login.py 71行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 68, in test_login_ok
    self.assertEqual(response.json()["status"],expected["status"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: 0 != 1
2024-04-15 18:53:41,977 py30 INFO test_login.py 58行：执行用例2：name为空
2024-04-15 18:53:41,978 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 18:53:41,978 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 18:53:41,978 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 18:53:41,978 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 18:53:41,978 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8942b"
}
2024-04-15 18:53:42,003 py30 INFO test_login.py 66行：用例的期望结果为：{
    "status": 0,
    "msg": "用户名不能为空"
}
2024-04-15 18:53:42,004 py30 ERROR test_login.py 71行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 69, in test_login_ok
    self.assertEqual(response.json()["msg"],expected["msg"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "D:\scb35\python\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
AssertionError: '系统异常，请联系管理员' != '用户名不能为空'
- 系统异常，请联系管理员
+ 用户名不能为空

2024-04-15 18:53:42,008 py30 INFO test_login.py 58行：执行用例3：word为空
2024-04-15 18:53:42,009 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 18:53:42,009 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 18:53:42,009 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 18:53:42,009 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 18:53:42,010 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":""
}
2024-04-15 18:53:42,029 py30 INFO test_login.py 66行：用例的期望结果为：{
    "status": 0,
    "msg": "密码不能为空"
}
2024-04-15 18:53:42,030 py30 ERROR test_login.py 71行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 69, in test_login_ok
    self.assertEqual(response.json()["msg"],expected["msg"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "D:\scb35\python\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
AssertionError: '系统异常，请联系管理员' != '密码不能为空'
- 系统异常，请联系管理员
+ 密码不能为空

2024-04-15 18:53:42,033 py30 INFO test_login.py 58行：执行用例4：name错误
2024-04-15 18:53:42,034 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 18:53:42,034 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 18:53:42,035 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 18:53:42,035 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 18:53:42,035 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin1",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b"
}
2024-04-15 18:53:42,058 py30 INFO test_login.py 66行：用例的期望结果为：{
    "status": 0,
    "msg": "用户名或密码错误"
}
2024-04-15 18:53:42,059 py30 ERROR test_login.py 71行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 69, in test_login_ok
    self.assertEqual(response.json()["msg"],expected["msg"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "D:\scb35\python\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
AssertionError: '系统异常，请联系管理员' != '用户名或密码错误'
- 系统异常，请联系管理员
+ 用户名或密码错误

2024-04-15 18:53:42,063 py30 INFO test_login.py 58行：执行用例5：word错误
2024-04-15 18:53:42,063 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 18:53:42,063 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 18:53:42,064 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 18:53:42,064 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 18:53:42,064 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b1"
}
2024-04-15 18:53:42,086 py30 INFO test_login.py 66行：用例的期望结果为：{
    "status": 1,
    "msg": "用户名或密码错误"
}
2024-04-15 18:53:42,086 py30 ERROR test_login.py 71行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 68, in test_login_ok
    self.assertEqual(response.json()["status"],expected["status"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: 0 != 1
2024-04-15 18:53:42,088 py30 INFO test_login.py 53行：======= 登录模块用例  执行结束========
2024-04-15 19:46:05,196 py30 INFO test_login.py 85行：======  登录模块用例 开始执行  ========
2024-04-15 19:46:05,197 py30 INFO test_login.py 95行：*********   执行用例1：正常登录   *********
2024-04-15 19:46:05,197 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:46:05,198 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:46:05,198 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:46:05,198 py30 INFO handle_requests.py 55行：请求url为：/token/get
2024-04-15 19:46:05,198 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b"
}
2024-04-15 19:46:05,212 py30 INFO test_login.py 95行：*********   执行用例2：name为空   *********
2024-04-15 19:46:05,212 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:46:05,212 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:46:05,212 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:46:05,212 py30 INFO handle_requests.py 55行：请求url为：/token/get
2024-04-15 19:46:05,213 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8942b"
}
2024-04-15 19:46:05,217 py30 INFO test_login.py 95行：*********   执行用例3：word为空   *********
2024-04-15 19:46:05,218 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:46:05,218 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:46:05,218 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:46:05,218 py30 INFO handle_requests.py 55行：请求url为：/token/get
2024-04-15 19:46:05,218 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":""
}
2024-04-15 19:46:05,222 py30 INFO test_login.py 95行：*********   执行用例4：name错误   *********
2024-04-15 19:46:05,222 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:46:05,222 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:46:05,222 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:46:05,223 py30 INFO handle_requests.py 55行：请求url为：/token/get
2024-04-15 19:46:05,223 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin1",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b"
}
2024-04-15 19:46:05,227 py30 INFO test_login.py 95行：*********   执行用例5：word错误   *********
2024-04-15 19:46:05,227 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:46:05,227 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:46:05,227 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:46:05,227 py30 INFO handle_requests.py 55行：请求url为：/token/get
2024-04-15 19:46:05,227 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b1"
}
2024-04-15 19:46:05,233 py30 INFO test_login.py 89行：======  登录模块用例 执行结束  ========
2024-04-15 19:47:24,528 py30 INFO test_login.py 85行：======  登录模块用例 开始执行  ========
2024-04-15 19:47:24,529 py30 INFO test_login.py 95行：*********   执行用例1：正常登录   *********
2024-04-15 19:47:24,530 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:47:24,530 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:47:24,530 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:47:24,530 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:47:24,530 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b"
}
2024-04-15 19:47:24,569 py30 INFO test_login.py 111行：用例的期望结果为：{
    "status": 1,
    "msg": "登录成功",
    "data": {
        "token": "eyJhbGciOiJIUzI1NiJ9.eyJqdGkiOiIxIiwic3ViIjoiYWRtaW4iLCJpc3MiOiJEUlpLIiwiaWF0IjoxNzEyMDI4OTA3LCJleHAiOjE3MTIwOTM3MDd9.LBQLzIE9jovM4md8psL46eV4J7M9NhHEIx5pyekqcuc",
        "loginStatus": "2"
    }
}
2024-04-15 19:47:24,569 py30 ERROR test_login.py 121行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 113, in test_register_ok
    self.assertEqual(response.json()["status"],expected["status"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: 0 != 1
2024-04-15 19:47:24,575 py30 INFO test_login.py 95行：*********   执行用例2：name为空   *********
2024-04-15 19:47:24,575 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:47:24,575 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:47:24,575 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:47:24,575 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:47:24,575 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8942b"
}
2024-04-15 19:47:24,596 py30 INFO test_login.py 111行：用例的期望结果为：{
    "status": 0,
    "msg": "用户名不能为空"
}
2024-04-15 19:47:24,596 py30 ERROR test_login.py 121行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 114, in test_register_ok
    self.assertEqual(response.json()["msg"], expected["msg"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "D:\scb35\python\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
AssertionError: '系统异常，请联系管理员' != '用户名不能为空'
- 系统异常，请联系管理员
+ 用户名不能为空

2024-04-15 19:47:24,598 py30 INFO test_login.py 95行：*********   执行用例3：word为空   *********
2024-04-15 19:47:24,599 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:47:24,599 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:47:24,599 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:47:24,600 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:47:24,600 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":""
}
2024-04-15 19:47:24,629 py30 INFO test_login.py 111行：用例的期望结果为：{
    "status": 0,
    "msg": "密码不能为空"
}
2024-04-15 19:47:24,630 py30 ERROR test_login.py 121行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 114, in test_register_ok
    self.assertEqual(response.json()["msg"], expected["msg"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "D:\scb35\python\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
AssertionError: '系统异常，请联系管理员' != '密码不能为空'
- 系统异常，请联系管理员
+ 密码不能为空

2024-04-15 19:47:24,635 py30 INFO test_login.py 95行：*********   执行用例4：name错误   *********
2024-04-15 19:47:24,635 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:47:24,635 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:47:24,635 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:47:24,635 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:47:24,635 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin1",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b"
}
2024-04-15 19:47:24,657 py30 INFO test_login.py 111行：用例的期望结果为：{
    "status": 0,
    "msg": "用户名或密码错误"
}
2024-04-15 19:47:24,658 py30 ERROR test_login.py 121行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 114, in test_register_ok
    self.assertEqual(response.json()["msg"], expected["msg"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "D:\scb35\python\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
AssertionError: '系统异常，请联系管理员' != '用户名或密码错误'
- 系统异常，请联系管理员
+ 用户名或密码错误

2024-04-15 19:47:24,660 py30 INFO test_login.py 95行：*********   执行用例5：word错误   *********
2024-04-15 19:47:24,661 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:47:24,661 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:47:24,661 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:47:24,661 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:47:24,661 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b1"
}
2024-04-15 19:47:24,681 py30 INFO test_login.py 111行：用例的期望结果为：{
    "status": 1,
    "msg": "用户名或密码错误"
}
2024-04-15 19:47:24,681 py30 ERROR test_login.py 121行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 113, in test_register_ok
    self.assertEqual(response.json()["status"],expected["status"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: 0 != 1
2024-04-15 19:47:24,684 py30 INFO test_login.py 89行：======  登录模块用例 执行结束  ========
2024-04-15 19:48:04,752 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:48:04,752 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:48:04,752 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:48:04,752 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:48:04,752 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b'}
2024-04-15 19:48:04,791 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:48:04,791 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8', 'accessToken': 'eyJhbGciOiJIUzI1NiJ9.eyJqdGkiOiIxIiwic3ViIjoiYWRtaW4iLCJpc3MiOiJEUlpLIiwiaWF0IjoxNzEzMTgxNjQ5LCJleHAiOjE3MTMyNDY0NDl9.a7IRwJfqg9WanyNXhIViTNUUEru64MB7lHlSfGRsFW8'}
2024-04-15 19:48:04,792 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:48:04,792 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/api/v1/visitor/create
2024-04-15 19:48:04,794 py30 INFO handle_requests.py 56行：请求数据为：{'carNo': '粤B921981', 'startTime': '2024-09-14 11:11:11', 'endTime': '2024-09-17 11:11:11', 'visitName': '张三', 'visitorPhone': '19852461254', 'visitorReason': '备注', 'dsn': '145456784321,456451544653', 'objectId': 16}
2024-04-15 19:48:30,426 py30 INFO test_login.py 85行：======  登录模块用例 开始执行  ========
2024-04-15 19:48:30,426 py30 INFO test_login.py 95行：*********   执行用例1：正常登录   *********
2024-04-15 19:48:30,427 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:48:30,427 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:48:30,427 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:48:30,427 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:48:30,427 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b"
}
2024-04-15 19:48:30,458 py30 INFO test_login.py 111行：用例的期望结果为：{
    "status": 1,
    "msg": "登录成功",
    "data": {
        "token": "eyJhbGciOiJIUzI1NiJ9.eyJqdGkiOiIxIiwic3ViIjoiYWRtaW4iLCJpc3MiOiJEUlpLIiwiaWF0IjoxNzEyMDI4OTA3LCJleHAiOjE3MTIwOTM3MDd9.LBQLzIE9jovM4md8psL46eV4J7M9NhHEIx5pyekqcuc",
        "loginStatus": "2"
    }
}
2024-04-15 19:48:30,458 py30 ERROR test_login.py 121行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 113, in test_register_ok
    self.assertEqual(response.json()["status"],expected["status"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: 0 != 1
2024-04-15 19:48:30,463 py30 INFO test_login.py 95行：*********   执行用例2：name为空   *********
2024-04-15 19:48:30,463 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:48:30,463 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:48:30,463 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:48:30,463 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:48:30,464 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8942b"
}
2024-04-15 19:48:30,483 py30 INFO test_login.py 111行：用例的期望结果为：{
    "status": 0,
    "msg": "用户名不能为空"
}
2024-04-15 19:48:30,484 py30 ERROR test_login.py 121行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 114, in test_register_ok
    self.assertEqual(response.json()["msg"], expected["msg"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "D:\scb35\python\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
AssertionError: '系统异常，请联系管理员' != '用户名不能为空'
- 系统异常，请联系管理员
+ 用户名不能为空

2024-04-15 19:48:30,486 py30 INFO test_login.py 95行：*********   执行用例3：word为空   *********
2024-04-15 19:48:30,486 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:48:30,486 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:48:30,486 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:48:30,486 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:48:30,486 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":""
}
2024-04-15 19:48:30,531 py30 INFO test_login.py 111行：用例的期望结果为：{
    "status": 0,
    "msg": "密码不能为空"
}
2024-04-15 19:48:30,532 py30 ERROR test_login.py 121行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 114, in test_register_ok
    self.assertEqual(response.json()["msg"], expected["msg"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "D:\scb35\python\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
AssertionError: '系统异常，请联系管理员' != '密码不能为空'
- 系统异常，请联系管理员
+ 密码不能为空

2024-04-15 19:48:30,534 py30 INFO test_login.py 95行：*********   执行用例4：name错误   *********
2024-04-15 19:48:30,534 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:48:30,534 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:48:30,534 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:48:30,534 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:48:30,534 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin1",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b"
}
2024-04-15 19:48:30,555 py30 INFO test_login.py 111行：用例的期望结果为：{
    "status": 0,
    "msg": "用户名或密码错误"
}
2024-04-15 19:48:30,556 py30 ERROR test_login.py 121行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 114, in test_register_ok
    self.assertEqual(response.json()["msg"], expected["msg"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "D:\scb35\python\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
AssertionError: '系统异常，请联系管理员' != '用户名或密码错误'
- 系统异常，请联系管理员
+ 用户名或密码错误

2024-04-15 19:48:30,558 py30 INFO test_login.py 95行：*********   执行用例5：word错误   *********
2024-04-15 19:48:30,558 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:48:30,558 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:48:30,559 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:48:30,559 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:48:30,559 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b1"
}
2024-04-15 19:48:30,576 py30 INFO test_login.py 111行：用例的期望结果为：{
    "status": 1,
    "msg": "用户名或密码错误"
}
2024-04-15 19:48:30,576 py30 ERROR test_login.py 121行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 113, in test_register_ok
    self.assertEqual(response.json()["status"],expected["status"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: 0 != 1
2024-04-15 19:48:30,579 py30 INFO test_login.py 89行：======  登录模块用例 执行结束  ========
2024-04-15 19:50:52,680 py30 INFO test_login.py 85行：======  登录模块用例 开始执行  ========
2024-04-15 19:50:52,682 py30 INFO test_login.py 95行：*********   执行用例1：正常登录   *********
2024-04-15 19:50:52,682 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:50:52,683 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:50:52,683 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:50:52,683 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:50:52,683 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b"
}
2024-04-15 19:50:52,710 py30 INFO test_login.py 111行：用例的期望结果为：{
    "status": 1,
    "msg": "登录成功"
    
}
2024-04-15 19:50:52,711 py30 ERROR test_login.py 121行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 113, in test_register_ok
    self.assertEqual(response.json()["status"],expected["status"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: 0 != 1
2024-04-15 19:50:52,721 py30 INFO test_login.py 95行：*********   执行用例2：name为空   *********
2024-04-15 19:50:52,722 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:50:52,722 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:50:52,722 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:50:52,722 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:50:52,722 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8942b"
}
2024-04-15 19:50:52,742 py30 INFO test_login.py 111行：用例的期望结果为：{
    "status": 0,
    "msg": "用户名不能为空"
}
2024-04-15 19:50:52,743 py30 ERROR test_login.py 121行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 114, in test_register_ok
    self.assertEqual(response.json()["msg"], expected["msg"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "D:\scb35\python\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
AssertionError: '系统异常，请联系管理员' != '用户名不能为空'
- 系统异常，请联系管理员
+ 用户名不能为空

2024-04-15 19:50:52,744 py30 INFO test_login.py 95行：*********   执行用例3：word为空   *********
2024-04-15 19:50:52,745 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:50:52,745 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:50:52,745 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:50:52,745 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:50:52,745 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":""
}
2024-04-15 19:50:52,776 py30 INFO test_login.py 111行：用例的期望结果为：{
    "status": 0,
    "msg": "密码不能为空"
}
2024-04-15 19:50:52,776 py30 ERROR test_login.py 121行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 114, in test_register_ok
    self.assertEqual(response.json()["msg"], expected["msg"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "D:\scb35\python\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
AssertionError: '系统异常，请联系管理员' != '密码不能为空'
- 系统异常，请联系管理员
+ 密码不能为空

2024-04-15 19:50:52,779 py30 INFO test_login.py 95行：*********   执行用例4：name错误   *********
2024-04-15 19:50:52,779 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:50:52,780 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:50:52,780 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:50:52,780 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:50:52,780 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin1",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b"
}
2024-04-15 19:50:52,882 py30 INFO test_login.py 111行：用例的期望结果为：{
    "status": 0,
    "msg": "用户名或密码错误"
}
2024-04-15 19:50:52,883 py30 ERROR test_login.py 121行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 114, in test_register_ok
    self.assertEqual(response.json()["msg"], expected["msg"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "D:\scb35\python\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
AssertionError: '系统异常，请联系管理员' != '用户名或密码错误'
- 系统异常，请联系管理员
+ 用户名或密码错误

2024-04-15 19:50:52,886 py30 INFO test_login.py 95行：*********   执行用例5：word错误   *********
2024-04-15 19:50:52,887 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:50:52,887 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:50:52,887 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:50:52,887 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:50:52,887 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b1"
}
2024-04-15 19:50:53,025 py30 INFO test_login.py 111行：用例的期望结果为：{
    "status": 1,
    "msg": "用户名或密码错误"
}
2024-04-15 19:50:53,025 py30 ERROR test_login.py 121行：断言失败！
Traceback (most recent call last):
  File "C:\Users\26360\PycharmProjects\PY99\day1\TestCases\test_login.py", line 113, in test_register_ok
    self.assertEqual(response.json()["status"],expected["status"])
  File "D:\scb35\python\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "D:\scb35\python\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: 0 != 1
2024-04-15 19:50:53,027 py30 INFO test_login.py 89行：======  登录模块用例 执行结束  ========
2024-04-15 19:55:14,138 py30 INFO test_login.py 85行：======  登录模块用例 开始执行  ========
2024-04-15 19:55:14,139 py30 INFO test_login.py 95行：*********   执行用例1：正常登录   *********
2024-04-15 19:55:14,139 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:55:14,140 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:55:14,140 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:55:14,140 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:55:14,140 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b"
}
2024-04-15 19:55:14,188 py30 INFO test_login.py 107行：登陆结果为：{'status': 0, 'msg': '系统异常，请联系管理员'}
2024-04-15 19:55:14,188 py30 INFO test_login.py 95行：*********   执行用例2：name为空   *********
2024-04-15 19:55:14,189 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:55:14,189 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:55:14,189 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:55:14,189 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:55:14,190 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8942b"
}
2024-04-15 19:55:14,230 py30 INFO test_login.py 107行：登陆结果为：{'status': 0, 'msg': '系统异常，请联系管理员'}
2024-04-15 19:55:14,231 py30 INFO test_login.py 95行：*********   执行用例3：word为空   *********
2024-04-15 19:55:14,232 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:55:14,232 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:55:14,232 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:55:14,233 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:55:14,233 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":""
}
2024-04-15 19:55:14,275 py30 INFO test_login.py 107行：登陆结果为：{'status': 0, 'msg': '系统异常，请联系管理员'}
2024-04-15 19:55:14,276 py30 INFO test_login.py 95行：*********   执行用例4：name错误   *********
2024-04-15 19:55:14,276 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:55:14,276 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:55:14,277 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:55:14,277 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:55:14,277 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin1",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b"
}
2024-04-15 19:55:14,432 py30 INFO test_login.py 107行：登陆结果为：{'status': 0, 'msg': '系统异常，请联系管理员'}
2024-04-15 19:55:14,433 py30 INFO test_login.py 95行：*********   执行用例5：word错误   *********
2024-04-15 19:55:14,433 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:55:14,433 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:55:14,434 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:55:14,434 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:55:14,434 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b1"
}
2024-04-15 19:55:14,466 py30 INFO test_login.py 107行：登陆结果为：{'status': 0, 'msg': '系统异常，请联系管理员'}
2024-04-15 19:55:14,466 py30 INFO test_login.py 89行：======  登录模块用例 执行结束  ========
2024-04-15 19:59:16,720 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 19:59:16,721 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 19:59:16,721 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 19:59:16,721 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 19:59:16,721 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b'}
2024-04-15 19:59:16,764 py30 INFO handle_requests.py 119行：登录的请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 20:00:36,075 py30 INFO test_login.py 85行：======  登录模块用例 开始执行  ========
2024-04-15 20:00:36,075 py30 INFO test_login.py 95行：*********   执行用例1：正常登录   *********
2024-04-15 20:00:36,076 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 20:00:36,076 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 20:00:36,076 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 20:00:36,076 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 20:00:36,076 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b"
}
2024-04-15 20:00:36,100 py30 INFO test_login.py 107行：登陆结果为：{'status': 0, 'msg': '系统异常，请联系管理员'}
2024-04-15 20:00:36,101 py30 INFO test_login.py 95行：*********   执行用例2：name为空   *********
2024-04-15 20:00:36,101 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 20:00:36,101 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 20:00:36,101 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 20:00:36,101 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 20:00:36,101 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8942b"
}
2024-04-15 20:00:36,115 py30 INFO test_login.py 107行：登陆结果为：{'status': 0, 'msg': '系统异常，请联系管理员'}
2024-04-15 20:00:36,115 py30 INFO test_login.py 95行：*********   执行用例3：word为空   *********
2024-04-15 20:00:36,115 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 20:00:36,115 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 20:00:36,115 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 20:00:36,115 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 20:00:36,115 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":""
}
2024-04-15 20:00:36,128 py30 INFO test_login.py 107行：登陆结果为：{'status': 0, 'msg': '系统异常，请联系管理员'}
2024-04-15 20:00:36,129 py30 INFO test_login.py 95行：*********   执行用例4：name错误   *********
2024-04-15 20:00:36,129 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 20:00:36,129 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 20:00:36,129 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 20:00:36,129 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 20:00:36,129 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin1",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b"
}
2024-04-15 20:00:36,141 py30 INFO test_login.py 107行：登陆结果为：{'status': 0, 'msg': '系统异常，请联系管理员'}
2024-04-15 20:00:36,141 py30 INFO test_login.py 95行：*********   执行用例5：word错误   *********
2024-04-15 20:00:36,141 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 20:00:36,141 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 20:00:36,141 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 20:00:36,141 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 20:00:36,141 py30 INFO handle_requests.py 56行：请求数据为：{
    "loginName":"admin",
    "pssWord":"7c4a8d09ca3762af61e59520943dc26494f8941b1"
}
2024-04-15 20:00:36,155 py30 INFO test_login.py 107行：登陆结果为：{'status': 0, 'msg': '系统异常，请联系管理员'}
2024-04-15 20:00:36,155 py30 INFO test_login.py 89行：======  登录模块用例 执行结束  ========
2024-04-15 20:04:20,511 py30 INFO test_login.py 85行：======  登录模块用例 开始执行  ========
2024-04-15 20:04:20,512 py30 INFO test_login.py 95行：*********   执行用例1：正常登录   *********
2024-04-15 20:04:20,512 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 20:04:20,512 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 20:04:20,513 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 20:04:20,513 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 20:04:20,513 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b'}
2024-04-15 20:04:20,550 py30 INFO test_login.py 109行：登陆结果为：{'status': 1, 'msg': '登录成功', 'data': {'token': 'eyJhbGciOiJIUzI1NiJ9.eyJqdGkiOiIxIiwic3ViIjoiYWRtaW4iLCJpc3MiOiJEUlpLIiwiaWF0IjoxNzEzMTgyNjI1LCJleHAiOjE3MTMyNDc0MjV9.k9Z_be87s_pe_p5Meee0LqJORV-lZHqS2FhM4bZ7Mwg', 'loginStatus': '2'}}
2024-04-15 20:04:20,550 py30 INFO test_login.py 95行：*********   执行用例2：name为空   *********
2024-04-15 20:04:20,551 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 20:04:20,551 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 20:04:20,551 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 20:04:20,551 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 20:04:20,552 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': '', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8942b'}
2024-04-15 20:04:20,574 py30 INFO test_login.py 109行：登陆结果为：{'status': 0, 'msg': '用户名不能为空'}
2024-04-15 20:04:20,575 py30 INFO test_login.py 95行：*********   执行用例3：word为空   *********
2024-04-15 20:04:20,575 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 20:04:20,575 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 20:04:20,575 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 20:04:20,576 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 20:04:20,576 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin', 'pssWord': ''}
2024-04-15 20:04:20,596 py30 INFO test_login.py 109行：登陆结果为：{'status': 0, 'msg': '密码不能为空'}
2024-04-15 20:04:20,597 py30 INFO test_login.py 95行：*********   执行用例4：name错误   *********
2024-04-15 20:04:20,597 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 20:04:20,597 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 20:04:20,598 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 20:04:20,598 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 20:04:20,598 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin1', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b'}
2024-04-15 20:04:20,631 py30 INFO test_login.py 109行：登陆结果为：{'status': 0, 'msg': '用户名或密码错误'}
2024-04-15 20:04:20,632 py30 INFO test_login.py 95行：*********   执行用例5：word错误   *********
2024-04-15 20:04:20,632 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-15 20:04:20,632 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-15 20:04:20,632 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-15 20:04:20,633 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-15 20:04:20,633 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b1'}
2024-04-15 20:04:20,655 py30 INFO test_login.py 109行：登陆结果为：{'status': 0, 'msg': '用户名或密码错误'}
2024-04-15 20:04:20,655 py30 INFO test_login.py 89行：======  登录模块用例 执行结束  ========
2024-04-16 18:40:59,512 py30 INFO test_login.py 85行：======  登录模块用例 开始执行  ========
2024-04-16 18:40:59,514 py30 INFO test_login.py 95行：*********   执行用例1：正常登录   *********
2024-04-16 18:40:59,514 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-16 18:40:59,514 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-16 18:40:59,515 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-16 18:40:59,515 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-16 18:40:59,515 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b'}
2024-04-16 18:40:59,817 py30 INFO test_login.py 109行：登陆结果为：{'status': 1, 'msg': '登录成功', 'data': {'token': 'eyJhbGciOiJIUzI1NiJ9.eyJqdGkiOiIxIiwic3ViIjoiYWRtaW4iLCJpc3MiOiJEUlpLIiwiaWF0IjoxNzEzMjY0MDI1LCJleHAiOjE3MTMzMjg4MjV9.UL-yyJoNEia1mm4p1XIW9TytcIW1JZifVdmSeCBwE_4', 'loginStatus': '2'}}
2024-04-16 18:40:59,818 py30 INFO test_login.py 95行：*********   执行用例2：name为空   *********
2024-04-16 18:40:59,819 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-16 18:40:59,819 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-16 18:40:59,820 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-16 18:40:59,820 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-16 18:40:59,820 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': '', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8942b'}
2024-04-16 18:40:59,842 py30 INFO test_login.py 109行：登陆结果为：{'status': 0, 'msg': '用户名不能为空'}
2024-04-16 18:40:59,842 py30 INFO test_login.py 95行：*********   执行用例3：word为空   *********
2024-04-16 18:40:59,843 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-16 18:40:59,843 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-16 18:40:59,844 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-16 18:40:59,844 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-16 18:40:59,844 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin', 'pssWord': ''}
2024-04-16 18:40:59,865 py30 INFO test_login.py 109行：登陆结果为：{'status': 0, 'msg': '密码不能为空'}
2024-04-16 18:40:59,865 py30 INFO test_login.py 95行：*********   执行用例4：name错误   *********
2024-04-16 18:40:59,866 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-16 18:40:59,866 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-16 18:40:59,866 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-16 18:40:59,867 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-16 18:40:59,868 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin1', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b'}
2024-04-16 18:40:59,918 py30 INFO test_login.py 109行：登陆结果为：{'status': 0, 'msg': '用户名或密码错误'}
2024-04-16 18:40:59,918 py30 INFO test_login.py 95行：*********   执行用例5：word错误   *********
2024-04-16 18:40:59,919 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-16 18:40:59,919 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-16 18:40:59,919 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-16 18:40:59,919 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-16 18:40:59,919 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b1'}
2024-04-16 18:40:59,972 py30 INFO test_login.py 109行：登陆结果为：{'status': 0, 'msg': '用户名或密码错误'}
2024-04-16 18:40:59,972 py30 INFO test_login.py 89行：======  登录模块用例 执行结束  ========
2024-04-18 17:05:50,927 py30 INFO test_login.py 84行：======  登录模块用例 开始执行  ========
2024-04-18 17:05:50,929 py30 INFO test_login.py 94行：*********   执行用例1：正常登录   *********
2024-04-18 17:05:50,930 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-18 17:05:50,930 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-18 17:05:50,931 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-18 17:05:50,931 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-18 17:05:50,931 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b'}
2024-04-18 17:05:51,156 py30 INFO test_login.py 108行：登陆结果为：{'status': 1, 'msg': '登录成功', 'data': {'token': 'eyJhbGciOiJIUzI1NiJ9.eyJqdGkiOiIxIiwic3ViIjoiYWRtaW4iLCJpc3MiOiJEUlpLIiwiaWF0IjoxNzEzNDMxMTUyLCJleHAiOjE3MTM0OTU5NTJ9.pQy2jN3AcYw7l4X5VmUXJpsqzBaDw-t2VXfgvdB-d_8', 'loginStatus': '2'}}
2024-04-18 17:05:51,156 py30 INFO test_login.py 94行：*********   执行用例2：name为空   *********
2024-04-18 17:05:51,157 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-18 17:05:51,157 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-18 17:05:51,158 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-18 17:05:51,158 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-18 17:05:51,158 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': '', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8942b'}
2024-04-18 17:05:51,182 py30 INFO test_login.py 108行：登陆结果为：{'status': 0, 'msg': '用户名不能为空'}
2024-04-18 17:05:51,182 py30 INFO test_login.py 94行：*********   执行用例3：word为空   *********
2024-04-18 17:05:51,183 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-18 17:05:51,183 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-18 17:05:51,184 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-18 17:05:51,184 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-18 17:05:51,184 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin', 'pssWord': ''}
2024-04-18 17:05:51,223 py30 INFO test_login.py 108行：登陆结果为：{'status': 0, 'msg': '密码不能为空'}
2024-04-18 17:05:51,224 py30 INFO test_login.py 94行：*********   执行用例4：name错误   *********
2024-04-18 17:05:51,224 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-18 17:05:51,225 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-18 17:05:51,225 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-18 17:05:51,225 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-18 17:05:51,226 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin1', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b'}
2024-04-18 17:05:51,247 py30 INFO test_login.py 108行：登陆结果为：{'status': 0, 'msg': '用户名或密码错误'}
2024-04-18 17:05:51,247 py30 INFO test_login.py 94行：*********   执行用例5：word错误   *********
2024-04-18 17:05:51,248 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-18 17:05:51,248 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-18 17:05:51,249 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-18 17:05:51,249 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-18 17:05:51,249 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b1'}
2024-04-18 17:05:51,301 py30 INFO test_login.py 108行：登陆结果为：{'status': 0, 'msg': '用户名或密码错误'}
2024-04-18 17:05:51,301 py30 INFO test_login.py 88行：======  登录模块用例 执行结束  ========
2024-04-18 17:06:41,763 py30 INFO test_login.py 84行：======  登录模块用例 开始执行  ========
2024-04-18 17:06:41,765 py30 INFO test_login.py 94行：*********   执行用例1：正常登录   *********
2024-04-18 17:06:41,766 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-18 17:06:41,767 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-18 17:06:41,768 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-18 17:06:41,769 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-18 17:06:41,770 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b'}
2024-04-18 17:06:41,815 py30 INFO test_login.py 108行：登陆结果为：{'status': 1, 'msg': '登录成功', 'data': {'token': 'eyJhbGciOiJIUzI1NiJ9.eyJqdGkiOiIxIiwic3ViIjoiYWRtaW4iLCJpc3MiOiJEUlpLIiwiaWF0IjoxNzEzNDMxMjAyLCJleHAiOjE3MTM0OTYwMDJ9.V76QVTEKM1JL33BjyEiW1sspZ6QorSZg4Ih6U20jGJI', 'loginStatus': '2'}}
2024-04-18 17:06:41,816 py30 INFO test_login.py 94行：*********   执行用例2：name为空   *********
2024-04-18 17:06:41,817 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-18 17:06:41,817 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-18 17:06:41,818 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-18 17:06:41,819 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-18 17:06:41,820 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': '', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8942b'}
2024-04-18 17:06:41,855 py30 INFO test_login.py 108行：登陆结果为：{'status': 0, 'msg': '用户名不能为空'}
2024-04-18 17:06:41,857 py30 INFO test_login.py 94行：*********   执行用例3：word为空   *********
2024-04-18 17:06:41,858 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-18 17:06:41,858 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-18 17:06:41,859 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-18 17:06:41,859 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-18 17:06:41,860 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin', 'pssWord': ''}
2024-04-18 17:06:41,897 py30 INFO test_login.py 108行：登陆结果为：{'status': 0, 'msg': '密码不能为空'}
2024-04-18 17:06:41,898 py30 INFO test_login.py 94行：*********   执行用例4：name错误   *********
2024-04-18 17:06:41,900 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-18 17:06:41,900 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-18 17:06:41,901 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-18 17:06:41,902 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-18 17:06:41,903 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin1', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b'}
2024-04-18 17:06:41,953 py30 INFO test_login.py 108行：登陆结果为：{'status': 0, 'msg': '用户名或密码错误'}
2024-04-18 17:06:41,954 py30 INFO test_login.py 94行：*********   执行用例5：word错误   *********
2024-04-18 17:06:41,955 py30 INFO handle_requests.py 45行：发起一次HTTP请求
2024-04-18 17:06:41,956 py30 INFO handle_requests.py 53行：请求头为：{'Content-Type': 'application/json;charset=UTF-8'}
2024-04-18 17:06:41,956 py30 INFO handle_requests.py 54行：请求方法为：POST
2024-04-18 17:06:41,957 py30 INFO handle_requests.py 55行：请求url为：http://192.168.66.67/ykt/token/get
2024-04-18 17:06:41,957 py30 INFO handle_requests.py 56行：请求数据为：{'loginName': 'admin', 'pssWord': '7c4a8d09ca3762af61e59520943dc26494f8941b1'}
2024-04-18 17:06:41,982 py30 INFO test_login.py 108行：登陆结果为：{'status': 0, 'msg': '用户名或密码错误'}
2024-04-18 17:06:41,983 py30 INFO test_login.py 88行：======  登录模块用例 执行结束  ========
